shard.yml

```diff
@@ -1,5 +1,6 @@
 name: placeos-models
-version: 4.10.0
+version: 4.10.1
+crystal: ~> 0.35
```

Comment:
Shouldn't this be ~> 0.36 ?

Reply:
either works, it still functions on 0.35